posthog/models/experiment.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
+    name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    filters: models.JSONField = models.JSONField(default=dict)
+    feature_flags: models.ManyToManyField = models.ManyToManyField("FeatureFlag", blank=False)
+    created_by: models.ForeignKey = models.ForeignKey("User", on_delete=models.CASCADE)
+    created_at: models.DateTimeField = models.DateTimeField(default=timezone.now)
+    updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
```

Comment:
It's hard to comment what's missing here without running into YAGNI as a lot of things are not ready?

Maybe let's not create the model yet in prod (DEBUG only) and add fields as we go along then do a single migration in prod?

Reply:
I am happy to leave this backend stuff off of this PR for now while we think about what we'd need for our model 👍